generated/nifake/session.py

```diff
@@ -955,6 +955,27 @@ def get_a_string_using_python_code(self, a_number):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return a_string_ctype.value.decode(self._encoding)
 
+    @ivi_synchronized
+    def get_a_string_with_specified_maximum_size(self, buffer_size):
```

Comment:
Why is a python user expected to pass in a buffer size?  That feels like a C API bleeding over into Python.  Is there really no way we can work around this?